scripts.js

```diff
@@ -133,7 +153,8 @@ function displayMap(year, firstLoad) {
             sleep(firstLoad ? 0 : 300)
         ])
             .then(() => buildInstitutionData(year))
-            .then((institutions) => placeMarkers(institutions)),
+            .then((institutions) => placeMarkers(institutions))
```

Comment:
Optional: drop paren around `(institutions)`

Reply:
I prefer this in this case.

Reply:
That's legti